src/main/java/net/dv8tion/jda/internal/entities/AbstractChannelImpl.java

```diff
@@ -166,8 +164,26 @@ public boolean isSynced()
     {
         if (getParent() == null)
             return true; // Channels without a parent category are always considered synced. Also the case for categories.
-        
-        return Helpers.deepEqualsUnordered(getParent().getPermissionOverrides(), getPermissionOverrides());
+
+        Stream<PermissionOverride> thisStream = getRolePermissionOverrides().stream().filter(
+                po -> po.getAllowedRaw() != 0 || po.getDeniedRaw() != 0
+        );
+        Stream<PermissionOverride> parentStream = getParent().getRolePermissionOverrides().stream().filter(
+                po -> po.getAllowedRaw() != 0 || po.getDeniedRaw() != 0
+        );
+        Collection<PermissionOverride> thisCheck = thisStream.collect(Collectors.toList());
+        Collection<PermissionOverride> parentCheck = parentStream.collect(Collectors.toList());
+
+        for(PermissionOverride check : thisCheck)
+        {
+            PermissionOverride matching = parentCheck.stream().filter(po -> check.getIdLong() == po.getIdLong()).findFirst().orElse(null);
```

Comment:
This seems rather inefficient when you have a map of id -> override at your disposal.

Reply:
I will look for an better option